appserver/microprofile/config/pom.xml

```diff
@@ -63,6 +63,13 @@
 
     <build>
         <plugins>
+            <plugin>
```

Comment:
Why is it required here? Also it is more code than before ...

Reply:
> Helidon 3:
> Java 17 minimum JDK. Java 11 no longer supported.

Needed, because otherwise - with inherited `release=11` - `java.lang.Record` is unavailable due to compiler checking what is and what isn't in given release.

Maybe it would be better to compile this module for release 17?